### PR TITLE
Bug/6364 remove opt out code when no tag is set

### PR DESCRIPTION
**Pull request by derweili**
## Summary

Addresses issue:

- #6364 

## Relevant technical choices

- Return early on `print_tracking_opt_out()` when `$property_id` empty so not opt-out code is printed.
- Add new test case for empty `propertyId` to `AnalyticsTest`.

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: includes/Modules/Analytics.php [hunk 1]
```diff
@@ -1157,6 +1157,10 @@ private function print_tracking_opt_out() {
 		$account_id  = $settings['accountID'];
 		$property_id = $settings['propertyID'];
 
+		if ( empty( $property_id ) ) {
```
Comment (aaemnnosttv) [inline includes/Modules/Analytics.php:1160]: Let's check this first before we call `is_tracking_disabled` above as there's no point in invoking all of that if there is no property ID 👍

Comment (derweili) [reply includes/Modules/Analytics.php:1157]: Updated as suggested.

**Review by aaemnnosttv — APPROVED** [review]
LGTM, thanks @derweili !
